core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js

```diff
@@ -320,6 +320,11 @@ var projectsNavProviderFn = function(context) {
   });
 };
 
+var customPageNotFoundHandler = function(wrongPath, wasAnyPathFitted) {
```

Comment:
Maybe it would be good to add to documentation that this function might receive `wrongPath` and `wasAnyPathFitted`?